php/Controllers/SettingsController.php

```diff
@@ -335,6 +335,7 @@ private function getProfile(): ?ProfileModel {
 
     /**
      * Get registered post types in WordPress
+     * TODO: only return public post_types? Can there be post_types that aren't public but can be handeld by Coyote?
```

Comment:
Good Q. I would say that's for the admin to decide: if the post type is not public (yet), images within them might still require descriptions, so a batch import should still post them to the API. If they're not public post types, the user-facing side won't be affected.

Reply:
updated the function to return all post_types instead of only public